drivers/bq2429x/Kconfig

```
@@ -0,0 +1,33 @@
+# Copyright (C) 2020 Locha Inc
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_BQ2429X
+    bool "BQ2429x"
+    depends on HAS_PERIPH_I2C
+    depends on HAS_PERIPH_GPIO
+    depends on TEST_KCONFIG
+    select MODULE_PERIPH_I2C
+    select MODULE_PERIPH_GPIO
+    default y if HAVE_BQ2429X
+
+config MODULE_BQ2429X_INT
+    bool "BQ2429x with interrupt functionality"
+    depends on HAS_PERIPH_GPIO_IRQ
+    select MODULE_PERIPH_GPIO_IRQ
+    select MODULE_BQ2429X
```

I think this should be a dependency instead, as we try to model things bottom-up
```suggestion
    depends on MODULE_BQ2429X
```

---

Fixed at 1148431af8 , also renamed added a little help message and reworded the symbol description.